protostar/utils/protostar_directory.py

```diff
@@ -42,15 +42,26 @@ def protostar_test_only_cairo_packages_path(self) -> Path:
         assert self.protostar_binary_dir_path is not None
         return self.protostar_binary_dir_path / "cairo"
 
-    def _get_runtime_constants(self) -> RuntimeConstantsDict:
+    def _read_runtime_constants(self) -> Optional[RuntimeConstantsDict]:
         constants_str = (
             self.info_dir_path / ProtostarDirectory.RUNTIME_CONSTANTS_FILE_NAME
         ).read_text("utf-8")
         return json.loads(constants_str)
 
-    def get_runtime_constant(self, name: RuntimeConstantName) -> RuntimeConstantValue:
+    def get_runtime_constant(
+        self, name: RuntimeConstantName
+    ) -> Optional[RuntimeConstantValue]:
         if self._runtime_constants is None:
-            self._runtime_constants = self._get_runtime_constants()
+            try:
+                self._runtime_constants = self._read_runtime_constants()
+            except FileNotFoundError as ex:
+                logging.getLogger().warning(
+                    "Couldn't load constant `%s` from %s", name, ex.filename
+                )
+                return None
+
+        if self._runtime_constants is None or name not in self._runtime_constants:
```

Comment:
the `name` is always  `in self._runtime_constants` - it's guaranteed by type safety

Reply:
[750cb6d](https://github.com/software-mansion/protostar/pull/918/commits/750cb6de3f899e0182b3da8515b07cf216480890)